tag_database

```diff
@@ -48,6 +48,7 @@ debounce:function
 decapitalize:string,array
 deepClone:object,recursion
 deepFlatten:array,recursion
+deepFreeze:object,recursion
```

Comment:
Maybe this fits under `advanced`? It's not very difficult, but the concepts are a bit more advanced.

Reply:
I wouldn't put it under advanced,  but you might be right too. 

TBH I see deepFreeze as an essential snippet when writing functional JS.

Reply:
Let's keep it as not `advanced` for now and we'll see what the feedback says.